flaml/model.py

```diff
@@ -658,53 +666,52 @@ def _compute_metrics_by_dataset_name(self, eval_pred):
             )
             return metric_dict
 
-    def predict_proba(self, X_test):
-        assert (
-            self._task in CLASSIFICATION
-        ), "predict_proba() only for classification tasks."
-
+    def init_model_for_predict(self, X_test):

```

Comment:
Is this method supposed to be called by external code? if not, use "_" in the beginning of the method name.